src/incubator/hooks/useHiddenLocation.ts

```diff
@@ -34,16 +35,30 @@ export default function useHiddenLocation<T extends View>(props: HiddenLocationP
 
   const [hiddenLocation, setHiddenLocation] = useState<HiddenLocation>(getHiddenLocation({}));
 
-  const onLayout = useCallback((event: LayoutChangeEvent) => {
-    const {width, height} = event.nativeEvent.layout;
-    if (containerRef.current && !hiddenLocation.wasMeasured) {
-      containerRef.current.measureInWindow((x: number, y: number) => {
-        setHiddenLocation(getHiddenLocation({x, y, width, height, wasMeasured: true}));
+  const measure = useCallback(() => {
+    if (!hiddenLocation.wasMeasured && containerRef.current) {
+      containerRef.current.measureInWindow((x, y) => {
+        if (!hiddenLocation.wasMeasured && size.current) {
```

Comment:
Is there a reason to check the `!hiddenLocation.wasMeasured` here again?

Reply:
From what I can tell this is not a synchronous call, and I prefer to be safe since this was not a consistent bug and I cannot truly verify the call stack that caused the bug (it felt very much like a race condition).